src/main/java/net/rptools/maptool/model/Token.java

```diff
@@ -2269,8 +2281,8 @@ public TokenTransferData toTransferData() {
     td.put(TokenTransferData.ASSET_ID, imageAssetMap.get(null));
     td.put(TokenTransferData.Z, z);
     td.put(TokenTransferData.SNAP_TO_SCALE, snapToScale);
-    td.put(TokenTransferData.WIDTH, scaleX);
-    td.put(TokenTransferData.HEIGHT, scaleY);
+    td.put(TokenTransferData.WIDTH, scaleX); // this makes no sense
```

Comment:
You are correct; adding this comment makes no sense at all, If you think its a bug test with the correct value to see result or add an issue